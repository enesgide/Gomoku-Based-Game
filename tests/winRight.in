place S1
place S2
history
place R1
place R2
history
place Q1
place Q2
place P1
place P2
place O1