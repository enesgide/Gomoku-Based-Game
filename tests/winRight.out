S1S2
S1S2R1R2
Black wins!
S1S2R1R2Q1Q2P1P2O1
Thank you for playing!
